src/test/kotlin/com/zepben/evolve/streaming/get/NetworkConsumerClientTest.kt

```diff
@@ -150,6 +150,32 @@ internal class NetworkConsumerClientTest {
         }
     }
 
+    @Test
+    internal fun `server receives linked container options`() {
```

Comment:
todo: This test is meant to be about the server receiving the equipment container flags, not validating that IdentifiedObject responses can be processed,  so there is no need to be looping over all of the possible valid response objects.

Since you do not validate anything against the response, you do not even need to call `resp.onNext`, and this test will be substantially shorter/clearer.

Can you also add a test to give coverage to the `getEquipmentForContainers` version that takes an `Iterable` (I thought I saw you add this elsewhere, but I can not see it anymore, so either I am imagining it, it was deleted again, or perhaps it was in another branch). You can probably combine this call with using the default argumnets for the included container types.